lib/fs/util.go

```diff
@@ -117,13 +117,13 @@ func SanitizePath(path string) string {
 	}
 
 	path = strings.TrimSpace(b.String())
-	if windowsIsReserved(path) {
+	if _, reserved := windowsIsReserved(path); reserved {
 		path = "-" + path
 	}
 	return path
 }
 
-func windowsIsReserved(part string) bool {
+func windowsIsReserved(part string) (string, bool) {
```

Comment:
Empty string could be enough to indicate not found imo.

Reply:
Yeah